pyuploadcare/api/client.py

```diff
@@ -123,6 +149,66 @@ def request(  # noqa: C901
                 else:
                     raise
 
+    def _handle_httpx_arguments(  # noqa: max-complexity: 6
+        self,
+        follow_redirects: typing.Optional[bool] = None,
+        allow_redirects: typing.Optional[bool] = None,
+    ) -> bool:
+        """
+        Encapsulate smooth updating for httpx dependency:
+         - using of `allow_redirects` is allowed,
+           but that argument will be deleted in the next major version
+         - using of `follow_redirects` is allowed when
+           `allow_redirects` is not set
+        """
+        redirecting = True
+
+        allow_redirects_is_set = allow_redirects is not None
+        follow_redirects_is_set = follow_redirects is not None
+
+        if allow_redirects_is_set and follow_redirects_is_set:
```

Comment:
these vars don't bring any value. and you don't use them in the next clause.

```suggestion
        if allow_redirects is not None and follow_redirects is not None:
```